src/js/api.js

```diff
@@ -171,6 +171,9 @@ $(function () {
         return ajax("GET", endpoint("/v1/admin/pending"))
     }
 
+		v1.admin.isOwner = function (cID){
+			return ajax("GET", endpoint("/v1/admin/" + cID + "/is-owner"))
```

Comment:
`isOwner` is, to me, a little too vague function name. Maybe something more verbose: `is-collection-owner`?